common/test/package.scala

```diff
@@ -65,6 +66,7 @@ trait TestSettings {
 trait ConfiguredTestSuite extends ConfiguredServer with ConfiguredBrowser with ExecutionContexts {
   this: ConfiguredTestSuite with org.scalatest.Suite =>
 
+  lazy val webClient = new WebClient(BrowserVersion.CHROME)
```

Comment:
if it's just a simulation that's fine, I just thought it was an actual chrome which would have been very heavy!  sounds fine to me...the only benefit of hitting the controller would be the static typing rather than using the routes, although depending if you're hoping to test the routes as well, maybe you want that! 😀